src/tests/erc20-pool-contracts.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('ERC20 Pool', () => {
     expect(bucketStatus.exchangeRate.eq(toWad(1))).toBe(true);
 
     const lpBalance = await bucket.lpBalance(signerLender.address);
-    expect(lpBalance).toEqual(quoteAmount);
+    expect(fromWad(lpBalance)).toEqual('9999.54337899543379');
```

Comment:
NIT: it'd be more readable if the number would be named variable, ie `expectedBalance`. Ideally with some explanation of number origin, when it makes sense.

below there is a change from 25k to 24998.858447488584475 which looks magical and fragile for external observer

Reply:
Maybe to do between range in such cases? Or if it should be exactly this number with max accuracy it'd great to understand how this number was chosen